meta-resin-common/recipes-connectivity/networkmanager/files/NetworkManager.conf

```diff
@@ -0,0 +1,6 @@
+[main]
+dns=default
+plugins=keyfile
+
+[keyfile]
+unmanaged-devices=interface-name:docker0
```

Comment:
I asked this a couple of times but I haven't received any feedback. Is there any reason not to blacklist tun devices too?